src/tests/compositor/Compositor_Resolution_001.test.js

```diff
@@ -0,0 +1,59 @@
+import {
+  pluginDeactivate,
+  pluginActivate,
+  setCompositorResolution,
+  getCompositorResolution,
+} from '../../commonMethods/commonFunctions'
+import constants from '../../commonMethods/constants'
+
+export default {
+  title: 'Compositor Resolution - 001',
+  description: 'Sets the resolution to 480i and checks whether the same resolution is set or not',
+  context: {
+    resolution: '480i',
+  },
+  steps: [
+    {
+      description: 'Deactivate Netflix Plugin and check deactivated or not',
+      test: pluginDeactivate,
+      params: constants.netFlixPlugin,
+      assert: 'deactivated',
```

Comment:
I wouldn't use Netflix for a test here.

Reply:
Updated code as per review comment and used Webkitbrowser